app/views/validation_requests/_time_extension_validation_requests.html.erb

```diff
@@ -1,15 +1,14 @@
 <li>
-  <h2 class="moj-task-list__section">
-    <span class="moj-task-list__section-number"><%= count_total_requests(@validation_requests, "time_extension_validation_requests") %></span>Confirm changes to your application expiry date
-  </h2>
   <ul class="moj-task-list__items" id="time-extension-validation-requests">
```

Comment:
I think I got rid of all references to the MOJ task list, e.g. _replacement_document_validation_requests.html.erb

Reply:
Oh... thanks for catching the regression. Will find the relevant class and update.